eos-phone-home

```diff
@@ -401,49 +400,49 @@ class PhoneHome(object):
     def _do_ping(self):
         if self._send_to_server(self.PING_ENDPOINT, self.PING_VARIABLES):
             if self._debug:
-                print("Debugging turned on so not incrementing count.")
+                log.info("Debugging turned on so not incrementing count.")
             else:
                 # increment the counter upon successful ping only
                 count = self._lookup_or_get_variable('count')
                 self._set_count(count + 1)
 
-            print("Pinged!")
+            log.info("Pinged!")
         else:
-            print("Could not send ping to server!", file=sys.stderr)
+            log.warning("Could not send ping to server!")
 
             return False
 
         return True
 
     def run(self):
         if not os.path.isdir(self.STATE_DIRECTORY):
-            print("State folder (%s) not found! Exiting!" %
-                  self.STATE_DIRECTORY, file=sys.stderr)
+            log.fatal("State folder (%s) not found! Exiting!",
```

Comment:
It's an alias for `logging.critical` it turns out. I'll use the documented name.